### DISCUSSION TITLE
Is it logical to make discord bot using deno?

### DISCUSSION THREAD

**Discussion by impixel980** [community/community#143869] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am willing to make a discord bot. previously I used discord.js but this time it's not working . i wanna make a bot using deno

**Comment by monwarqx1250** [comment]
something wrong with you. I already used discordjs to make my bot . and it's working fine. did you installed discordjs. install it by deno install npm:discordjs if you don't

**Comment by ebndev** [comment]
Hi @impixel980 & @monwarqx1250, thanks for being a part of the GitHub Community!

While we appreciate you wanting to contribute to Discussions, please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. 

Thanks for understanding and we hope to see you participate in other discussions in the future!
